pkg/verify/tlog_test.go

```diff
@@ -52,6 +52,47 @@ func TestTlogVerifier(t *testing.T) {
 	assert.Error(t, err)
 }
 
+type goodAndUntrustedLogEntry struct {
```

Comment:
Minor: I updated the wording on https://github.com/sigstore/sigstore-go/pull/45/files that you could consider adopting here: `oneTrustedOneUntrustedTimestampEntity`, `UntrustedTestEntity`, etc.

Reply:
done